packages/trpc/server/routers/viewer/teams/getUpgradeable.handler.ts

```diff
@@ -2,22 +2,19 @@ import { IS_TEAM_BILLING_ENABLED } from "@calcom/lib/constants";
 import { prisma } from "@calcom/prisma";
 import { MembershipRole } from "@calcom/prisma/enums";
 import { teamMetadataSchema } from "@calcom/prisma/zod-utils";
-import type { TrpcSessionUser } from "@calcom/trpc/server/trpc";
 
 type GetUpgradeableOptions = {
```

Comment:
This only needs userId so lets make it a lot more testable by only passing in userId